pack/Km.json

```diff
@@ -39,6 +39,45 @@
         "traits": "House Bolton. The North.",
         "type_code": "location"
     },
+
+    {
+        "code": "08045",
+        "cost": 2,
+        "deck_limit": 3,
+        "faction_code": "thenightswatch",
+        "illustrator": "Stephen Somers",
+        "is_intrigue": true,
+        "is_loyal": false,
+        "is_military": true,
+        "is_power": false,
+        "is_unique": false,
+        "name": "Haunted Forest Scout",
+        "octgn_id": null,
+        "pack_code": "Km",
+        "position": 45,
+        "quantity": 3,
+        "strength": 2,
+        "text": "No attachments except <i>Weapon</i>.\nHaunted Forest Scout cannot be declared as a defender.\nWhile you control 3 or more <i>Ranger</i> characters, Haunted Forest Scout does not kneel when declared as an attacker.",
+        "traits": "Ally. Ranger.",
+        "type_code": "character"
+    },
+    {
+        "code": "08046",
+        "cost": 1,
+        "deck_limit": 3,
+        "faction_code": "thenightswatch",
+        "illustrator": "",
```

Comment:
It's somewhat illegible on the card image but based on previous artists I believe this reads as "Paolo Puggioni". If not, it can be corrected when the pack is actually released.